app/src/main/res/layout-land/activity_focus_parent.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rearrange and reformat the code. Remove hardcoding and unnecessary extra blank lines.